aiidalab_qe/steps.py

```diff
@@ -631,6 +711,9 @@ def submit(self, _=None):
         if not run_bands:
             builder.pop("bands")
 
+        if not run_pdos:
+            builder.pop("pdos")
```

Comment:
Do we actually need the `run_pdos` (and the `run_bands`) variable? And do we need to remove them from the `parameters` dict? This would also work, right?
```suggestion
        if not parameters["run_pdos"]:
            builder.pop("pdos")
```

Reply:
Will try whether not pop in from parameters works.